spec/orms/ar.rb

```diff
@@ -99,8 +141,97 @@ class Genre < ActiveRecord::Base
   json_schema { { name: String } }
 end
 
-3.times { Genre.create(name: ('a'..'z').to_a.sample(8).join) }
-3.times { Author.create(name: ('a'..'z').to_a.sample(8).join) }
-3.times { Book.create(name: ('a'..'z').to_a.sample(8).join, genre_id: Genre.all.sample.id) }
-3.times { Publisher.create(name: ('a'..'z').to_a.sample(8).join, book_id: Book.all.sample.id) }
-3.times { Award.create(name: ('a'..'z').to_a.sample(8).join, book_id: Book.all.sample.id) }
+class Executive < ActiveRecord::Base
+  has_one :assistant
+
+  include Surrealist
+
+  json_schema { { name: String, assistant: Object } }
+end
+
+class Assistant < ActiveRecord::Base
+  belongs_to :executive
+
+  include Surrealist
+
+  json_schema { { name: String, executive: Executive } }
+end
+
+class Manager < ActiveRecord::Base
+  has_many :employees
+  belongs_to :executive
+
+  include Surrealist
+
+  json_schema { { name: String } }
+end
+
+class Employee < ActiveRecord::Base
+  belongs_to :manager
+
+  include Surrealist
+
+  json_schema { { name: String, manager: Manager } }
+end
+
+class PromKing < ActiveRecord::Base
+  belongs_to :prom_couple
+  has_one :prom, through: :prom_couple
+
+  include Surrealist
+
+  json_schema { { name: String, prom: Object } }
```

Comment:
I suggest having different columns for different classes here, so specs can be more understandable. (something like `prom_king_name`, `prom_name` would do). It's a bit hard to debug right now.

Reply:
Will do